### PR TITLE
implement zipper

### LINKED ISSUE #736: zipper: implement exercise
There are exercises that are not yet implemented in Python track (the full list of them - https://github.com/exercism/python/issues/417#issuecomment-334841778).
You can help us change this situation by implementing this one. Ref #417

The guide ["Implement an exercise from specification"](https://github.com/exercism/docs/blob/master/you-can-help/implement-an-exercise-from-specification.md) will help you succeed.

If you still have questions, I encourage you to ask them before starting working on the issue.

I also suggest claiming the issue by creating an empty PR as stated [here](https://github.com/exercism/docs/blob/master/you-can-help/implement-an-exercise-from-specification.md#avoiding-duplicate-work)

Please note that there are some special ['closing issue' keywords](https://help.github.com/articles/closing-issues-using-keywords/) that you can put in description of your PR and Github will take care of closing the corresponding issue automatically after merge.
This simple thing helps us a lot, and makes our life a bit easier

### PR DESCRIPTION
**Pull request by cmccandless**
Resolves #736 

TODO:

- [x] add README
- [x] update `config.json`
- [x] write template solution (`zipper.py`)
- [x] write test cases in `zipper_test.py`
- [x] write example solution (`example.py`)

### CODE REVIEW

File: exercises/zipper/zipper.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    @staticmethod
+    def from_tree(tree):
+        return Zipper(tree)
+
+    def __init__(self, root, parent=None):
+        self.root = dict(root)
+        self.parent = None if parent is None else dict(parent)
+
+    def value(self):
+        return self.root['value']
```
Comment (m-a-ge) [inline exercises/zipper/zipper.py]: Should `pass` here I guess

File: exercises/zipper/zipper_test.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import unittest
+
+from zipper import Zipper
+
+
```
Comment (m-a-ge) [inline exercises/zipper/zipper_test.py]: Could you please also leave a comment stating what version of `canonical-data.json` the tests were adopted as discussed in #784 if aplicable?
The format is:
```text
# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0
```

File: exercises/zipper/zipper.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+class Zipper(object):
+    @staticmethod
+    def from_tree(tree):
+        return Zipper(tree)
+
+    def __init__(self, root, parent=None):
+        self.root = dict(root)
```
Comment (m-a-ge) [inline exercises/zipper/zipper.py]: `pass`

**Comment by cmccandless** [comment]
@m-a-ge Requested changes made.

**Comment by m-a-ge** [comment]
Thanks, @cmccandless 👍

**Pull request merged**